metador_core/widget/common.py

```diff
@@ -221,7 +229,14 @@ class Plugin(FileWidget.Plugin):
 
     @overrides
     def show(self) -> Viewable:
-        return pn.pane.Audio(self.file_url(), name=self.title)
+        # TODO check how to set type value from file metadata
+        return pn.pane.HTML(
+            f"""
+                <audio controls>
+                    <source src={self.file_url()} type="audio/mp3">
+                </audio>
+            """,
+        )
```

Comment:
Please complete the TODO, I think the MIME type shoul be accessible in `self._meta.encodingFormat`